lib/Rules.js

```diff
@@ -23,8 +24,37 @@ Rules.prototype.run = function(data) {
       throw new Error("rules must have an `if` and `then` object");
     }
 
-    if (this.testsMatch(rule["if"], data)) {
-      this.runOutcomes(rule.then, outcomeResults);
+    if (_.isPlainObject(rule["if"])) {
+
+      if (this.testsMatch(rule["if"], data)) {
+        this.runOutcomes(rule.then, outcomeResults);
+      }
+      else {
+        if (rule.elseThen) {
+          this.runOutcomes(rule.elseThen, outcomeResults);
+        }
+      }
+    }
+    else if (_.isArray(rule["if"])) { // can track N number of rule sets, i.e. series of OR conditions
+
+      var trackFailedValidation = [];
+
+      for (var j = 0; j < rule["if"].length; j++) {
+        var item = rule["if"][j];
+        if (!this.testsMatch(item, data)) {
+          trackFailedValidation.push(true);
```

Comment:
Instead of looping through all items in the array, we can save on performance by breaking:
Could rename `trackFailedValidation` to `testFailed` and just:
```
testFailed = true;
break;
```